### ISSUE TITLE
Can't change CP settings by using  controller 

### ISSUE THREAD

**Issue by gruszon18** [Courseplay/Courseplay_FS22#73]
Hello!  CP Is working. Not perfectly but still. Im using version 7.0.0.3.    In game menu i think CP settings should show in one "tab" not in 3.. I noticed i cant check or change any settings by using controller, only by mouse ( im using Xbox series  X controller ).  This in game interface of worker complicates stuff a lot .. I found hard  to set an worker to work again on next point when he has been blocked by something..  You guys probably know what to add in latter versions, but bring back the  color of the CP routes, its helps a lot.  Its nice touch, when you have course generated, you dont have to open generator or interface to hire CP worker,  You just press "H" .  Its more friendly to "controller players like me. And i miss "setting" About type of field work, or i just didn't notice it yet

**Comment by Tensuko** [comment]
As far as I know, this is work in progress.

But we have to split it into 3 tabs, there is no way arround it.

**Comment by gruszon18** [comment]
Okay, thanks for the response! Dunno if someone else report it, but I noticed that combine, if he gets to 100% unloading auger is bugging, my English is pretty bad, so forgive me, "when unloading pipe is folded, starts to move to unfold but just a little and then its closing" and its doing that very fast, back and forth till you don't stop the worker and manually open the auger to unload. I will try to record it when it happens again.

**Comment by Tensuko** [comment]
The pipeproblem in your video is already fixed.
Generate a new course, clear the old one or do you change the targetposition at the job menu ?
